tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/PublicAssetSetMethodContract.scala

```diff
@@ -30,6 +33,12 @@ import reactor.core.scala.publisher.SMono
 import scala.jdk.CollectionConverters._
 
 object PublicAssetSetMethodContract {
+  val CONFIGURATION: JmapRfc8621Configuration = JmapRfc8621Configuration(
+    urlPrefixString = URL_PREFIX_DEFAULT,
+    websocketPrefixString = WEBSOCKET_URL_PREFIX_DEFAULT,
+    publicAssetQuotaLimit = PublicAssetQuotaLimit.of(Size.of(500L, Size.Unit.B)).get
```

Comment:
Is correct when put `publicAssetQuotaLimit` in `JmapRfc8621Configuration` ?
I'm not sure, IMO "public asset" is not one of "rfc8621"
WDYT?

Reply:
Agree it should not leak in ASF James code